lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+  def initialize
+    hello
+    dishes
+  end
+
+  def dishes
+    @dishes =
+      { name: 'Burger', price: 10.95 },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.